### DISCUSSION TITLE
Request installation tutorial for Windows

### DISCUSSION THREAD

**Discussion by Longhuiberkeley** [borglab/gtsam#1295] [Q&A]
Hello, is there any tutorial/ step-by-step guide on how to install GTSAM, on Windows 11 with MATLAB wrap? As someone who is not familiar with compilers, cmake, or boost, I have tried to install the package for a few days now 🥲. 

Or alternatively, can someone give me some suggestions on what version of softwares I have higher chance of success (which ones I use the latest ones, which one I should use the old ones for stability?)
I have tried some combo of: cmake=3.24.0rc/ 3.24.2, vs=2017/2022; gtsam=3.2.3?/4.1.1/develop; boost=1.65/1.80. 

While following this was helpful https://github.com/borglab/gtsam/blob/develop/INSTALL.md#windows-installation , I still fail 🥲. Appreciate any advice, thanks in advance!

**Comment by wang-rich** [comment]
Windows10使用VS2019安装gtsam的教程网上有的

**Reply by XQYY** [reply]
你好，请问您可以给我一些指点吗，我失败了很多次，一直没有成功，真的很感谢

**Comment by Oussamaaat** [comment]
Hello, i'm looking for a tutorial as well any news after 2 years ?
